tests/test_all_fields.py

```diff
@@ -247,6 +243,12 @@ def getPKsToRecordsDict(self, stream, records, duplicates=False):  # BUG_9720
         return pks_to_record_dict_1, pks_to_record_dict_2
 
 
+    def epoch_to_dt(self, epoch_ts):
```

Comment:
We have one "time converter" in base.py as well, Can we use that?

Reply:
Yes, updated it.